OrbitGl/FrameTrack.cpp

```diff
@@ -18,20 +18,32 @@ constexpr const uint64_t kHeightCapAverageMultipleUint64 = 6;
 constexpr const float kBoxHeightMultiplier = 3.f;
 }  // namespace
 
+float FrameTrack::GetMaximumScaleFactor() const {
+  float scale_factor = 0.f;
+  if (stats_.average_time_ns() > 0) {
+    scale_factor =
+      static_cast<double>(stats_.max_ns()) / static_cast<double>(stats_.average_time_ns());
```

Comment:
You should make scale_factor double (as the variable "radio" was before), or make both of these static_cast<float> instead of double. I think that float is enough here.

Reply:
Agree, not sure why it was such a mess before. Trying with float for now.